airflow/cli/commands/celery_command.py

```diff
@@ -28,6 +28,7 @@
 from celery import maybe_patch_concurrency  # type: ignore[attr-defined]
 from celery.app.defaults import DEFAULT_TASK_LOG_FMT
 from celery.signals import after_setup_logger
+from deprecated import deprecated
```

Comment:
NIT: We need a comment at the beginnint of the file starting with DO NOT MODIFY THIS FILE unless it is a serious bugfix - all the new celery commands should be added in celery provider - and some explanation on why we have this file. 

Also I think we should add generic warning raised at the top-level module , this module should not be imported at all if new celery provider will be used.

https://stackoverflow.com/questions/30093490/how-to-declare-a-module-deprecated-in-python

Reply:
+1